pkg/dependency/parser/gradle/lockfile/parse.go

```diff
@@ -49,7 +49,7 @@ func (Parser) Parse(r xio.ReadSeekerAt) ([]types.Library, []types.Dependency, er
 			// There is no reliable way to determine direct dependencies (even using other files).
 			// Therefore, we mark all dependencies as Indirect.
 			// This is necessary to try to guess direct dependencies and build a dependency tree.
```

Comment:
nit: I think we can remove this comment now.

Reply:
Deleted https://github.com/aquasecurity/trivy/pull/6563/commits/8575199c265cd6abb03f54f6f3e27b6bde9867d7